README.md

```diff
@@ -0,0 +1,49 @@
+OSIC Reference implementation
+=============================
+
+# What is it?
+
+Reference implementation is opinionated way to deploy OpenStack. One of big
+qualities of OpenStack is it's flexibility, modularity and amount of underlying
+technilogies it can work with. But that also means that it has steep learning
+curve as sheer amount of options it allows is overwhelming (some services can
+have hundreds of them). We want to ease the entry to OpenStack by providing
+what is in our opinion good way to deploy OpenStack. We make sure that it's
+possible to be consumed by majority of users because it will be based on
+opensource as much as it is reasonable. In short, it's OpenStack authors would
+deploy:)
+
+# Design assumptions
+
+RefImpl will be designed to optimally run generic web services/application
+workload on a medium scale cloud (20 nodes to start with plans to expand
+to 300). Main users of it will be companies that are willing to deploy an
+OpenStack private cloud, but have no or minimum OpenStack experience, only
+experience as proficient operators. We don't assume any technology they
+are using.
```

Comment:
I think for us to have an opinion on how the cloud should scale and operate we have to have an opinion on the tech they are using.